consensus/consortium/v2/consortium.go

```diff
@@ -555,6 +553,10 @@ func (c *Consortium) verifyCascadingFields(chain consensus.ChainHeaderReader, he
 		if err := misc.VerifyEIP4844Header(parent, header); err != nil {
 			return err
 		}
+	} else {
+		if header.BlobCommitments != nil || header.BlobGasUsed != nil || header.ExcessBlobGas != nil {
```

Comment:
With blob commitments, I think we should check len(header.BlobCommitments)  != 0

Reply:
I don't test it yet but looking at the code: https://github.com/axieinfinity/ronin/blob/6feb34d9f01183f5b635868816aecad6a0526b37/rlp/decode.go#L310-L313
When rlp decoding, if the field is a slice and empty, it set the slice to 0-length slice not a nil. So I think check length is safer here

Reply:
I think you are right, fixed